lib/connectors/memory.js

```diff
@@ -723,6 +725,62 @@ Memory.prototype.updateAttributes = function updateAttributes(model, id, data, o
   }
 };
 
+Memory.prototype.replace = function replace(model, id, data, options, cb) {
```

Comment:
Then I believe I need to change the names in other PRs for connector, mysql and mongodb as well